### ISSUE TITLE
Autocomplete inherited method's signature

### ISSUE THREAD

**Issue by staticf0x** [python-lsp/python-lsp-server#458]
I don't know if this is pylsp or jedi issue, apologies if this is the wrong place.

Suppose I have this code:

```py
class A:
    def example(self, text: str) -> str:
        return text

class B(A):
    # Cursor here
```

when I start typing `def exam...`, autocomplete is shown, but the code filled in that place is then:

```py
class B(A):
    def example(self, text)
    #           ^^^^ self is selected
```

Would it be possible to also fill the method signature (typing) and perhaps a colon and move the cursor into the method body?

**Comment by ccordoba12** [comment]
I agree this would be quite nice to have when overriding a method in a subclass. But I think you should ask in the [Jedi repo](https://github.com/davidhalter/jedi) about it because Pylsp is simply a wrapper for it that makes it work with the LSP protocol.
